### PR TITLE
[#IC-425]  middleware for allowing ADVANCED message handling

### PR DESCRIPTION
**Pull request by gquadrati**
This PR depends on #199 

#### List of Changes

 - Add middleware for  allowing ADVANCED message handling

#### Motivation and Context

Advanced message handling

#### How Has This Been Tested?

Unit test, integration test

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: __integrations__/index.test.ts [hunk 1]
```diff
@@ -318,6 +336,57 @@ describe("Create Message", () => {
...
+
+      expect(resultGet.status).toEqual(200);
+      const detail = (await resultGet.json()) as MessageResponseWithContent;
+
+      expect(detail).toEqual(
+        expect.objectContaining({
+          message: expect.objectContaining({
+            id: messageId,
+            ...body.message,
+            feature_type: "STANDARD"
```
Comment (michaeldisaro) [inline __integrations__/index.test.ts:382]: Am I wrong or should this be in ADVANCED feature type?

Comment (gquadrati) [reply __integrations__/index.test.ts:382]: message model is not yet implemented, this will be changed :D

Comment (michaeldisaro) [reply __integrations__/index.test.ts]: Ok, I'll review this when the new model will be merged.

**Review by michaeldisaro — APPROVED** [review]
lgtm

**Review by AleDore — APPROVED** [review]
LGTM

**Pull request closed without merging**